source/_components/device_tracker.ubee.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Ubee Router"
+description: "Instructions on how to integrate Ubee routers into Home Assistant."
+date: 2018-12-26 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ubee.png
+ha_category: Presence Detection
+ha_release: pre 0.85
```

Comment:
Version number will need changed

Reply:
Thanks @DubhAd . 
What version should I use here?
I wrote this component for 0.85 and tested using that HA version.

Reply:
You have to put the release version there, so when this component is added to HA

Reply:
Thanks.
According to documentation, I need to put next release version. Since current version is 0.86, I put 0.87.

Reply:
Think you can bump again to 0.89 😆

Reply:
Done.
Maybe I should bump to 0.100 at least? :)
In the next few days I'm going to replace my Ubee router, which this device_tracker component I wrote for, with the one that can run DDWRT or OpenWRT. After that I will not be able to test this component anymore. I didn't expect to wait this long for my PR being merged. The good thing is that I'm using this component for more than 2 months w/o issues. :)